include/Mixer.h

```diff
@@ -366,6 +367,8 @@ class LMMS_EXPORT Mixer : public QObject
 
 	void clearInternal();
 
+	//! Function to be called before processing audio
+	//! Blocks if there are currently changes done in model
```

Comment:
Actually, it's called after processing and before writing the buffer to the audio device or a file.
Also, grammars are a bit odd in the last sentence.